media/css/firefox/feedback.less

```diff
@@ -0,0 +1,271 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import "../sandstone/lib.less";
+
+@font-face {
+    font-family: 'FA-Icons-Contact';
+    src: url('/media/fonts/icons-contact.eot?#iefix') format('embedded-opentype'),
+         url('/media/fonts/icons-contact.woff') format('woff'),
+         url('/media/fonts/icons-contact.ttf') format('truetype');
+    font-weight: normal;
+    font-style: normal;
+}
+
+main {
+    padding: 0 20px;
```

Comment:
(gonna go off on a tangent here so pardon the lengthy comment, maybe this should be a separate discussion elsewhere)

I'm not sure I like the idea of using <code>@baseLine</code> as a stand-in for <code>20px</code> every time we need a 20px value. The variable is named "baseline" because it's the common baseline for regular body text (14px with a 20px baseline). Keeping a standard baseline helps maintain a consistent vertical rhythm across the site. It really makes the most sense to use that variable for vertical spacing, especially with typography like the vertical space between paragraphs (which should really use em units proportional to the text anyway), or vertical space between two stacked boxes.

As with all variables, the point of defining <code>@baseLine</code> is that we can change it in one place and it will update everywhere it's used as a global control for vertical rhythm. I don't think changing that variable should change the padding on the left and right sides of this particular box.

BUT, if I'm going to use a variable here it would be <code>@gridGutterWidth</code>, which is more suited to horizontal spacing and also happens to be 20px so the end result is the same. So I think I'll do that. I could use <code>@baseLine</code> to pad the top and bottom of the box, except this box doesn't have top and bottom padding (but the next one does).

So as a general rule we should really only use <code>@baseLine</code> for vertical spacing where it won't cause width problems if its value ever changes. And sometimes 20px should just be 20px. We can use <code>@gridGutterWidth</code> for horizontal spacing, and if that one ever changes the <code>@gridColumnWidth</code> should also change accordingly. (Of course, so many pages have deviated from sandstone's grid system by now that I would dread the fallout of changing any of these variables at all... but that really is a separate discussion.)